src/httpserver.cpp

```diff
@@ -518,7 +518,7 @@ static void httpevent_callback_fn(evutil_socket_t, short, void* data)
         delete self;
 }
 
-HTTPEvent::HTTPEvent(struct event_base* base, bool deleteWhenTriggered, const boost::function<void(void)>& handler):
+HTTPEvent::HTTPEvent(struct event_base* base, bool deleteWhenTriggered, const std::function<void(void)>& handler):
```

Comment:
These removals in `httpserver.[cpp,h]` were implemented upstream as part of bitcoin/bitcoin#8421. Fine to remove here, any conflict should be simple to resolve.